debezium-connector-jdbc/src/main/java/io/debezium/connector/jdbc/transforms/ConvertCloudEventToSaveableForm.java

```diff
@@ -1,266 +0,0 @@
-/*
```

Comment:
@rkudryashov You might want to have a look at the removal of the ConvertCloudEventToSaveableForm SMT.

The result of the JDBC sink is now aligned with the conventional output of (sink) connectors. Basically it extracts the real payload (flattened or Debezium Envelope) from the `data` field of the CloudEvents message. Similar to handling the Debezium Envelope.
If you need a different structure of your events (that SMT felt like it solves your specific problem and that should not be in the core repo) please use other SMT combinations like ReplaceField for renaming etc or a custom SMT, or stream processing, to re-structure events like you need them.

Still I appreciate your feedback!